lib/reek/smells/uncommunicative_method_name.rb

```diff
@@ -45,21 +39,31 @@ def self.default_config
       #
       # @return [Array<SmellWarning>]
       #
-      # :reek:TooManyStatements: { max_statements: 9 }
-      def examine_context(ctx)
-        reject_names = value(REJECT_KEY, ctx, DEFAULT_REJECT_SET)
-        accept_names = value(ACCEPT_KEY, ctx, DEFAULT_ACCEPT_SET)
-        name = ctx.name.to_s
-        return [] if accept_names.include?(ctx.full_name)
-        var = name.gsub(/^[@\*\&]*/, '')
-        return [] if accept_names.include?(var)
-        return [] unless reject_names.find { |patt| patt =~ var }
+      def examine_context(context)
+        name = context.name.to_s
+        return [] if acceptable_name?(context: context, name: name)
```

Comment:
I would put the `name` argument first for clarity (not that it matters to Ruby, of course).